src/projects/models.py

```diff
@@ -6,6 +7,16 @@
 from django.urls import reverse
 
 from model_utils import Choices
+from django_resized import ResizedImageField
+
+
+def image_upload_to(instance, filename):
+    """Rename uploaded files with the object's slug."""
+
+    _, extension = splitext(filename)
+    name = instance.slug
+    filename = "projects/{}_image{}".format(name, extension)
```

Comment:
Utiliser une f-string
```suggestion
    filename = f"projects/{name}_image{extension}"
```